rest.getinsomnia.Insomnia.desktop

```diff
@@ -0,0 +1,7 @@
+[Desktop Entry]
+Name=Insomnia
+Exec=Insomnia
+Type=Application
+Icon=rest.getinsomnia.Insomnia.png
+Categories=Development;WebDevelopment
+Name[en_US]=rest.getinsomnia.Insomnia.desktop
```

Comment:
That's surely wrong name for any language.

Reply:
This is like the 4th package doing that, is there some bad template out there people are copying?

Reply:
I am not sure from where this translation comes from. Probably my text editor / file manager appends it somehow. Of course this is redundant.

This is my first Flathub package where I use Postman as template (https://github.com/flathub/com.getpostman.Postman).